stylecop.ruleset

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Microsoft Managed Recommended Rules" Description="These rules focus on the most critical problems in your code, including potential security holes, application crashes, and other important logic and design errors. It is recommended to include this rule set in any custom rule set you create for your projects." ToolsVersion="10.0">
+  <Rules AnalyzerId="StyleCop.Analyzers" RuleNamespace="StyleCop.Analyzers">
+    <!-- SA0001XmlCommentAnalysisDisabled -->
+    <Rule Id="SA0001" Action="None" />
+    <!-- SA1101PrefixLocalCallsWithThis -->
+    <Rule Id="SA1101" Action="None" />
+    <!-- SA1122UseStringEmptyForEmptyStrings -->
+    <Rule Id="SA1122" Action="None" />
+    <!-- SA1127GenericTypeConstraintsMustBeOnOwnLine -->
+    <Rule Id="SA1127" Action="None" />
+    <!-- SA1308VariableNamesMustNotBePrefixed -->
+    <Rule Id="SA1308" Action="None" />
+    <!-- SA1309FieldNamesMustNotBeginWithUnderscore -->
+    <Rule Id="SA1309" Action="None" />
+    <!-- SA1311StaticReadonlyFieldsMustBeginWithUpperCaseLetter -->
+    <Rule Id="SA1311" Action="None" />
```

Comment:
>None [](start = 30, length = 4)

enable?

Reply:
This ruleset is based on https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md